doc/user/rpki.rst

```diff
@@ -131,35 +131,28 @@ The following commands are available for independent of a specific cache server.
 
    The default value is 600 seconds.
 
-.. clicmd:: rpki cache (A.B.C.D|WORD) PORT [SSH_USERNAME] [SSH_PRIVKEY_PATH] [KNOWN_HOSTS_PATH] [source A.B.C.D] preference (1-255)
+.. clicmd:: rpki cache tcp HOST PORT [source A.B.C.D] preference (1-255)
 
+   Add a TCP cache server to the socket.
 
-   Add a cache server to the socket. By default, the connection between router
-   and cache server is based on plain TCP. Protecting the connection between
-   router and cache server by SSH is optional. Deleting a socket removes the
-   associated cache server and terminates the existing connection.
+.. clicmd:: rpki cache ssh HOST PORT SSH_USERNAME SSH_PRIVKEY_PATH [SERVER_PUBKEY] [source A.B.C.D] preference (1-255)
```

Comment:
[KNOWN_HOSTS_PATH] was the correct description

https://github.com/FRRouting/frr/blob/95b1b1d3e36066d73a1324bda1541b42e1f3c1a2/bgpd/bgp_rpki.c#L1501
https://github.com/rtrlib/rtrlib/blob/3e4f635a2f25df8b5da00f5788b4ae15e6234dd0/rtrlib/transport/ssh/ssh_transport.c#L75

Reply:
Fixed to known hosts file everywhere in our code. Thanks.